integration-tests/tests/src/tests/mixed.ts

```diff
@@ -135,7 +135,7 @@ const CollectionsOfMixedSchema: ObjectSchema = {
 };
 
 const bool = true;
-const int = 123;
+const int = BigInt(123);
```

Comment:
Yeah TS will see this as `bigint`, but the primary reason for this change is: Since these values will be used on a property defined as `"mixed"` in the schema, all regular numbers (e.g. `123`) that we pass will be interpreted as doubles (unlike if we would set `123` to a property defined as `int` in the schema). So in order to test the [@type == 'int'](https://github.com/realm/realm-js/blob/e4d7f1b9904d56ca1cb1a2d0e8d0b2fa84d6444b/integration-tests/tests/src/tests/mixed.ts#L2049) in queries, we can use `BigInt` to pass it as an `int64` to Core.